src/theme/colors.js

```diff
@@ -82,4 +82,72 @@ const colors = {
   ...getJson('COLORS')
 }
 
+const colors = {
+  default: '#ffffff',
+  overlay: '#ffffff',
+  hover: '#f6f8f7',
+  alert: '#ebf6e5',
+  defaultInverted: '#111111',
+  overlayInverted: '#191919',
+  divider: '#dadddc',
+  text: '#282828',
+  textSoft: '#979797',
+  textInverted: '#f0f0f0',
+  textInvertedSoft: '#828282',
```

Comment:
Probably a completed detail, but I'd change to `textSoftInverted`, to keep the `-Inverted` suffix consistent throughout.